<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Takes Screenshot]]>
    </line>
  </reporter-output>
  <suite started-at="2022-12-02T15:32:25 IST" name="Default suite" finished-at="2022-12-02T15:32:41 IST" duration-ms="15954">
    <groups>
    </groups>
    <test started-at="2022-12-02T15:32:25 IST" name="Default test" finished-at="2022-12-02T15:32:41 IST" duration-ms="15954">
      <class name="neostoxTest.NewTest">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:neostoxTest.NewTest@27406a17]" started-at="2022-12-02T15:32:25 IST" name="beforeClass" finished-at="2022-12-02T15:32:31 IST" duration-ms="5467" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:neostoxTest.NewTest@27406a17]" started-at="2022-12-02T15:32:31 IST" name="beforeMethod" finished-at="2022-12-02T15:32:33 IST" duration-ms="2137" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="validateuser()[pri:0, instance:neostoxTest.NewTest@27406a17]" started-at="2022-12-02T15:32:33 IST" name="validateuser" finished-at="2022-12-02T15:32:41 IST" duration-ms="8084" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [hi poonam biradar] but found [Hi poonam biradar]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [hi poonam biradar] but found [Hi poonam biradar]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at neostoxTest.NewTest.validateuser(NewTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Takes Screenshot]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateuser -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:neostoxTest.NewTest@27406a17]" started-at="2022-12-02T15:32:41 IST" name="afterClass" finished-at="2022-12-02T15:32:41 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:neostoxTest.NewTest@27406a17]" started-at="2022-12-02T15:32:41 IST" name="afterMethod" finished-at="2022-12-02T15:32:41 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- neostoxTest.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
